README.md

```diff
@@ -37,6 +37,13 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd .
   - `brew install redis`
   - Follow post-install instructions to enable Redis on startup. Otherwise,
     launch it manually with `brew services start redis`.
+1. Install ClamAV
+  - `brew install clamav`
+  - `cd` into the directory listed by the post-install instructions (and replace '${post_install_dir}' with it below), then:
+    - `touch clamd.sock`
+    - `echo "LocalSocket ${post_install_dir}" > clamd.conf` 
+    - `echo "DatabaseMirror database.clamav.net" > freshclam.conf`
+    - `freshclam -v`
```

Comment:
@omgitsbillryan do you have clamd.conf.sample and freshclam.conf.sample files in `/Users/vhaisaryanw/.brew/Cellar/clamav/0.99.2_1`? (mine are in `${homebrew_dir}/etc/clamav`)